.changeset/many-pumpkins-double.md

```diff
@@ -0,0 +1,8 @@
+---
+'@clerk/agent-toolkit': patch
+'@clerk/backend': patch
```

Comment:
You'll want to split the changesets so that the `@clerk/backend` changelog only says something about the small internal change